Makefile

```diff
@@ -68,7 +68,7 @@ env:
 
 
 node_modules/.installed : src/js/package.json
-	cd src/js && npm install --save-dev
```

Comment:
Maybe keep this as an install as it's not only going to be run from CI?

Reply:
I tried `npm install --save-dev` (without specifying any npm package) and its effect is the same as `npm install` which install dependencies and devDependencies. I think only `npm install --save-dev` somePackage` only makes the difference.

Reply:
I think the question becomes, among `npm install` or `npm ci`, which one is better for local dev installation? I did some search, and some people recommend using `npm ci` to install locked dependencies (after `git clone`) and only use `npm install`  to add or update some packages in the local environment. Not everyone says so but I have not found someone against this.